dmriprep/data/tests/bvec

```diff
@@ -0,0 +1,3 @@
+0.000000000000000000e+00 -4.900384695298463239e-02 -7.261448803568887334e-01 6.832152467094113613e-01 -8.447360612148637760e-01 7.303634462442597508e-01 5.099678730359662182e-02 1.799958601428245167e-02 4.441819297579839287e-01 0.000000000000000000e+00 9.895082430076975744e-01 4.698200933497754805e-01 -4.118960093847884307e-01 5.519743337902043301e-01 1.229201892472143037e-01 8.483882264407188734e-01 3.411076364416253948e-01 -3.610025270265337927e-01 0.000000000000000000e+00 4.719905602831905300e-01 8.560779066344169630e-01 -7.969294748620739943e-01 -4.670873535034215651e-01 -1.300449382906106351e-02 -8.824492810552004851e-01 -1.700067153978899667e-02 4.420322695335568985e-01 0.000000000000000000e+00 -3.650675437439669935e-01 -9.770420137099036229e-01 4.061086485971938109e-01 6.271361033032277765e-01 3.539851329366184296e-01 6.582818049316773168e-01 -4.229953470767730539e-01 -2.119635454050159407e-01 0.000000000000000000e+00 -9.115607335641323639e-01 3.110841596493176264e-01 -6.788842600996385368e-01 -1.349580345758559508e-01 6.469877073503395026e-01 -9.040953870952732618e-01 7.572927502328845595e-01 -1.430136584566050473e-01 0.000000000000000000e+00 -2.330642180393236174e-01 -6.638055334638701677e-01 -1.570514427726418805e-01 8.947830414154962497e-01 -5.937515669429782594e-01 5.998632467678286125e-03 -6.652272138969456128e-01 2.770963077032987942e-01 0.000000000000000000e+00 9.622964329590593113e-01 1.330403838856942245e-01 -7.897449535633485063e-01 1.939680948723744580e-01 2.359970500553113526e-01 8.844064781865559333e-01 -4.631901786104944962e-01 -7.000430539717695799e-01 0.000000000000000000e+00 2.000220036306656346e-01 -5.496906112542929712e-01 6.697194463156821431e-01 -2.370055696963325187e-01 -9.603842305537075852e-01 -4.900384695298463239e-02 -7.261448803568887334e-01 6.832152467094113613e-01 0.000000000000000000e+00 -8.447360612148637760e-01 7.303634462442597508e-01 5.099678730359662182e-02 1.799958601428245167e-02 4.441819297579839287e-01 9.895082430076975744e-01 4.698200933497754805e-01 -4.118960093847884307e-01 0.000000000000000000e+00 5.519743337902043301e-01 1.229201892472143037e-01 8.483882264407188734e-01 3.411076364416253948e-01 -3.610025270265337927e-01 4.719905602831905300e-01 8.560779066344169630e-01 -7.969294748620739943e-01 0.000000000000000000e+00 -4.670873535034215651e-01 -1.300449382906106351e-02 -8.824492810552004851e-01 -1.700067153978899667e-02 4.420322695335568985e-01 -3.650675437439669935e-01 -9.770420137099036229e-01 4.061086485971938109e-01 0.000000000000000000e+00 6.271361033032277765e-01 3.539851329366184296e-01 6.582818049316773168e-01 -4.229953470767730539e-01 -2.119635454050159407e-01
```

Comment:
An image-coordinates bvec file, consistent with the rasb `dwi.tsv` that is in RAS+ coordinates, using the affine of the fake nifti I also added (`dmriprep/data/tests/dwi.nii.gz`).

Reply:
Yeah I do think it's good to have at least one of these living in data